experimental/dds/tree2/src/shared-tree/sharedTree.ts

```diff
@@ -179,7 +179,7 @@ export class SharedTree
 		local: boolean,
 		localOpMetadata: unknown,
 	) {
-		if (!this.storedSchema.tryHandleOp(message.contents)) {
+		if (!this.storedSchema.tryHandleOp(message.contents as JsonCompatibleReadOnly)) {
```

Comment:
Don't want to use `as any` here since that would exactly match what was there before?
(`as any` is uglier in code and preferred where there might be stink. In other words, I think this should be visited in future as debt.)

Reply:
@CraigMacomber Do you or another tree expert know what the best type to use here is? I can leave a TODO or other follow up if needed too, but it'd be easier to fix it now.

Reply:
I think the proper fix is bigger with narrowing from possibilities similar to what was done recently in container runtime.
But cool if there is a simple, robust answer already.

Reply:
Updated this to `as any` and added a TODO.